# specify the namespace of the robot
robot_name: "oreo"

oreo:
  type: "Robot"
  name: "oreo"
  frame: "/base_link"

  # the sensors param namespace
  sensors: ["velodyne_vlp_16", "rgbcamera", "sick_laser", "hokuyo", "depthcamera", "fake_camera"]

  # uncomment this line to get ground-truth semantic label
  # sensors: ["velodyne_vlp_16", "rgbcamera", "sick_laser", "hokuyo", "depthcamera", "semantic_camera", "fake_camera"]

  translation: [0.0, 0.0, 0.5]
  rotation: [0.0, 0.0, 0.0, 1.0]

  # dynamic properties
  dynamic:
    mode: "dynamic" # dynamic: physical movement, legacy: override velocity
    friction_coefficient: 0.01
    angular_damping: 0.1
    linear_damping: 0.1
    standing_force: 100.0 # constantly apply force downward to prevent falling

  # the first name is the package name
  model_path: "habitat_ros/robot/oreo/oreo"

  cmd_topic: "/cmd_vel"

  # switch to true to publish ground-truth tf between map and base_link
  publish_transfromation: false

  odom:
    topic: "/odom"
    frame: "/odom"
    child_frame: "/base_link"
    
  tf_links:
    # base_link:
    #   frame: "/base_link"
    #   position: [0.0, 0.0, 0.0]
    #   orientation: [0.0, 0.0, 0.0, 1.0]

    base_footprint:
      frame: "/base_footprint"
      position: [0.0, 0.0, 0.0]
      orientation: [0.0, 0.0, 0.0, 1.0]